web_development_101/tying_it_all_together/conclusion.md

```diff
@@ -4,8 +4,40 @@ You should be fairly comfortable typing commands into your computer's command li
 
 With what you know already, you could start putting together some basic websites or strike out to increase your knowledge on your own.  However, we're here to help you get a lot further than that, to the point where you can confidently build a full-featured and scalable web application, whether on your own or working as a developer in a top-tier tech company.  
 
-You've come a long way up the learning curve but there is so much more cool stuff that you'll learn to do!
+You've come a long way up the learning curve but there is so much more cool stuff that you'll learn to do, and now is the time to decide which path you want to take. Either Ruby with Ruby on Rails or JavaScript with Node/Express. Both paths will guide you in building skills, building up a portfolio of projects and becoming a great developer. There is no right answer here on what is the next best path. Learning a different language can help you become a more well rounded developer, but sticking with one the whole way through could make you more of an expert in the lanaguage itself. Both paths will help you become a great developer. The important thing is to not switch halfway though, this will only set you back.
```

Comment:
"in the **lanaguage** itself" becomes "in the **language** itself"

Reply:
Fixed, thanks!